### ISSUE TITLE
Undefined method `camelize` for "goal":String

### ISSUE THREAD

**Issue by lamnk** [Fosome/garb#39]
My test file:

```
require 'rubygems'
require 'garb'
username = '***'
password = '***'
Garb::Session.login(username, password)
Garb::Account.all
```

Error:

```
/home/lamnk/.rvm/gems/ruby-1.8.7-p302/gems/garb-0.7.6/lib/garb.rb:36:in `to_ga': undefined method `camelize' for "goal":String (NoMethodError)
from /home/lamnk/.rvm/gems/ruby-1.8.7-p302/gems/garb-0.7.6/lib/garb/profile.rb:12:in `initialize'
from /home/lamnk/.rvm/gems/ruby-1.8.7-p302/gems/garb-0.7.6/lib/garb/profile.rb:24:in `new'
from /home/lamnk/.rvm/gems/ruby-1.8.7-p302/gems/garb-0.7.6/lib/garb/profile.rb:24:in `all'
from /home/lamnk/.rvm/gems/ruby-1.8.7-p302/gems/garb-0.7.6/lib/garb/profile.rb:24:in `map'
from /home/lamnk/.rvm/gems/ruby-1.8.7-p302/gems/garb-0.7.6/lib/garb/profile.rb:24:in `all'
from /home/lamnk/.rvm/gems/ruby-1.8.7-p302/gems/garb-0.7.6/lib/garb/account.rb:14:in `all'
from test.rb:10
```

**Comment by ryansmith3136** [comment]
I bet you are using active support 3. 

I also think tony fixed this issue. Hopefully he updates soon. 
http://twitter.com/tpitale/status/23942614330

**Comment by tpitale** [comment]
This should be fixed in 0.8.0 which supports ruby 1.9.2 and AS3. Sorry about that folks.
